### PR TITLE
Added Depth First Search in Python.

### PR DESCRIPTION
**Pull request by Pranav016**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added Graph traversal algorithm of Depth First Search in Python.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1528

### CODE REVIEW

**Comment by shraddhavp** [comment]
can you forment it using http://pep8online.com/ ?

File: Python/graphs/DFS.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    def getPathDFS(self, sv, ev):
+        visited = [False for i in range(self.vertices)]
+        hasPath, path = self.getPathDFShelper(sv, ev, visited)
+        if hasPath == True:
+            return path
+        return list()
+
+
+if __name__ == "__main__":
+    v, e = map(int, input().split())
```
Comment (atarax665) [inline Python/graphs/DFS.py:78]: Please format input and output by adding proper prompts for the user. Update sample I/O accordingly.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/graphs/DFS.py (hunk 1 above)
Comment (Pranav016) [reply Python/graphs/DFS.py:78]: Ok

Comment (Pranav016) [reply Python/graphs/DFS.py:78]: Done! @atarax665 😄

**Review by charlie219 — APPROVED** [review]
LGMT

**Comment by Pranav016** [comment]
Can you please review this? @iamrajiv @atarax665 @shraddhavp
Thanks

**Comment by shraddhavp** [comment]
LGTM !

**Review by iamrajiv — APPROVED** [review]

**Comment by Pranav016** [comment]
Ready to merge? @shraddhavp

**Pull request merged**